mayavi/core/base.py

```diff
@@ -10,7 +10,7 @@
 from copy import deepcopy
 import os
 import logging
-import imp
+import importlib
```

Comment:
You also need to change the places where `imp` is used; I see one use in `__module_view_default`.

Reply:
Thank you for your check. It's my fault.